plugins/signals-backend/src/service/router.test.ts

```diff
@@ -38,6 +39,10 @@ const discovery: jest.Mocked<PluginEndpointDiscovery> = {
   getExternalBaseUrl: jest.fn(),
 };
 
+const userInfo: jest.Mocked<UserInfoService> = {
```

Comment:
You can use `mockServices` from backend-test-utils for this too

Reply:
Yeah, though those were not exported yet